rust_src/src/editfns.rs

```diff
@@ -152,3 +155,36 @@ pub fn goto_char(position: LispObject) -> LispObject {
     };
     position
 }
+
+/// Insert COUNT (second arg) copies of BYTE (first arg).
```

Comment:
This seems a pretty poor docstring: saying the order, or which args are required, is already clear from the signature.

This is a good candidate for making better as part of #262. I don't think it's worth cleaning up in this PR.

Reply:
I added a TODO note.